cloud-nio/cloud-nio-spi/src/main/scala/cloud/nio/spi/HashType.scala

```diff
@@ -0,0 +1,47 @@
+package cloud.nio.spi
+
+import java.nio.{ByteBuffer, ByteOrder}
+import java.security.MessageDigest
+import java.util.Base64
+import java.util.zip.CRC32C
+
+object HashType extends Enumeration {
+  type HashType = Value
+
+  // crc32c as a hex string
+  val Crc32c: HashType.Value = Value("crc32c")
+  // AWS etag
```

Comment:
TOL I leave this to your discretion but I'm wondering if given the conversation the other day the variable shouldn't be something like 'AwsEtag' to differentiate from alternative versions

Reply:
👍  The reason I had for _not_ doing this was not a good one: the string value of these enum values corresponded to the checksum keys from the DRS response. I hadn't noticed that coupling and I don't like it, so thank you for pointing this out so I could fix it.